main.js

```diff
@@ -23,6 +24,11 @@ let win
 // a global reference of the logger object.
 let logger
 
+// global reference for electron store
+// keep all user hidden, app specific options here
+let electronStore = new Store()
+global.sharedObj = {electronStore: electronStore}
```

Comment:
Not needed

Reply:
what if we want to define multiple instances of it?

Reply:
Seems electron-store does not handle that! 
And they do not use IPC although it's mentioned that can be used in both processes.
I don't recommend using electron or electron.remote on the same file. we need to separate them in method (the reason is we will not use electron for mobile platforms and it will not be good to add many checks for that)

Reply:
electron-store handles that through defining different names to the config files saved.
so essentially I can't import or use libs from electron in the Renderer side?

Reply:
Ok, let's create a file to do electron remote calls instead of putting it in react component. we need to keep electron away from react components as possible

Reply:
why not we use the store in react (store.js) to save this config flag ? * 'show-welcome'

Reply:
I want to keep the app directory away from electron stuff. This is better to separate thing(e.g if we want to replace electron with other framework, it would be better to keep electron stuff (api) in one place, so it could be changed one time.

Reply:
I understand. what I was proposing in my comment above is to handle this persistence issue in react side using redux store.

Reply:
Let’s keep it away from React and Redux too

Reply:
okay so if I am keeping it away from electron stuff, and I am keeping it away from React and Redux too. where should I implement it :sweat_smile: ?